docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -2,9 +2,33 @@
 
 You can use Bluetooth functionalities in your application, such as managing the local Bluetooth adapter, bonding, and exchanging data between Bluetooth-enabled devices. The Bluetooth standard provides a peer-to-peer (P2P) data exchange functionality over short distance between compliant devices.
 
-This feature is supported in mobile and wearable applications only.
+<!-- TODO #00:
+      + add feature: http://tizen.org/feature/network.bluetooth.le.gatt.server
+      + check if there are some missing features in section
+ -->
+## Prerequisites
+
+To use the Application (in [mobile](../../api/latest/device_api/mobile/tizen/application.html), [wearable](../../api/latest/device_api/wearable/tizen/application.html) and [tv](../../api/latest/device_api/tv/tizen/application.html applications) and Bluetooth (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html) applications) APIs, the application has to request permission by adding the following privileges to the `config.xml` file:
 
-The main features of the Bluetooth API include:
+```
+<tizen:privilege name="http://tizen.org/privilege/application.launch"/>
+<tizen:privilege name="http://tizen.org/privilege/bluetooth"/>
+```
+<!-- END #00 -->
+
+<!-- TODO #01
+      - write about handling UUIDs, binary data
+      - write about toByteArray()
+	   - write about toDOMString()
+	   - write about toUint8Array()
+	   - write about uuidTo128bit()
+	   - write about uuidToShortestPossible()
+	   - write about uuidsEqual()
+ -->
+## Handling uuids and binary data in Bluetooth API
```

Comment:
Please, change "uuids" to "UUIDs" - UUID is an acronym and is usually written upper case.